http-builder-ng-core/src/test/groovy/groovyx/net/http/JavaHttpPatchSpec.groovy

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright (C) 2017 Keith Suderman
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */package groovyx.net.http
+
+import groovyx.net.http.tk.HttpPutTestKit
+
+/**
+ * @author Keith Suderman
```

Comment:
We don't use the `author` tags and prefer to use `git blame` to track authorship as needed. I am going to start a "contributors" section of the User Guide for making note of our recent contributors (with a link to their GitHub profile).